projects/kubernetes-sigs/cluster-api-provider-cloudstack/GIT_TAG

```diff
@@ -1 +1 @@
-v0.4.9-rc3
```

Comment:
Isn't it going to generate the shasum as the release number if you do this?

Reply:
Add your comment there: https://github.com/aws/eks-anywhere-build-tooling/pull/1719
I'll sync this branch accordingly.